nevergrad/optimization/test_nsga.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pylint: disable=redefined-outer-name,unsubscriptable-object,unused-variable,unused-import
+def test_doc_multiobjective() -> None:
+    # DOC_MULTIOBJ_0
+    import nevergrad as ng
+    from nevergrad.functions import MultiobjectiveFunction
+    import numpy as np
+
+    f = MultiobjectiveFunction(multiobjective_function=lambda x: [np.sum(x**2), np.sum((x - 1)**2)], upper_bounds=[2.5, 2.5])
+    print(f(np.array([1.0, 2.0])))
+
+    optimizer = ng.optimizers.NSGAIIES(parametrization=3, budget=100)  # 3 is the dimension, 100 is the budget.
+    recommendation = optimizer.minimize(f)
+
+    # The function embeds its Pareto-front:
+    print("My Pareto front:", [x[0][0] for x in f.pareto_front()])
+
+    # It can also provide a subset:
+    print("My Pareto front:", [x[0][0] for x in f.pareto_front(2, subset="random")])
+    print("My Pareto front:", [x[0][0] for x in f.pareto_front(2, subset="loss-covering")])
+    print("My Pareto front:", [x[0][0] for x in f.pareto_front(2, subset="domain-covering")])
+    # DOC_MULTIOBJ_1
+    assert len(f.pareto_front()) > 1
+    assert len(f.pareto_front(2, "loss-covering")) == 2
+    assert len(f.pareto_front(2, "domain-covering")) == 2
```

Comment:
all this test is a copy paste of a test which is only aiming at providing documentation, it should not be duplicated. Also, it does not actually test the feature you have added because the MultiobjectiveFunction is actually converting multiobjective to scalar using a hypervolume (what you added should work for the new way of dealing with multiobjtive, without this MultiobjectiveFunction layer)
You can remove this file entirely, I can deal with adding a test later on.